pkg/image/validate.go

```diff
@@ -4,24 +4,26 @@ import (
 	"strings"
 )
 
+type FileExt string
```

Comment:
how does a custom type help here (vs a string)?

Reply:
Just as a means of distinguishing these constants.  It's functionally the same as using "string" but more descriptive of their purpose.  I'm fine with reverting it though.